_includes/contact-us/contact-us.html

```diff
@@ -0,0 +1,63 @@
+{% comment %}
+<!--
+  ~ Copyright (c) 2022. Implex
+  ~ All rights reserved
+  -->
+{% endcomment %}
+
+<section class="section section-dark contact-us">
+    {% assign content = site.data.contact-us %}
+    <div class="content">
+        <div class="text-center">
+            <h1 class="transite transite-up sn-1">{{ content.title }}</h1>
+            <h2 class="transite transite-up sn-2">{{ content.subTitle }}</h2>
+        </div>
+        <form id="contact-form" class="form-wrapper transite transite-up sn-3">
+            <div class="input-wrapper">
+                <input id="name-input" name="name" placeholder="Your name" type="text">
+                <input id="email-input" name="email" placeholder="Your E-mail" type="text">
+            </div>
+            <label>
+                <textarea id="comment-input" name="comment" placeholder="Your text" rows="10"></textarea>
+            </label>
+            <div class="file-wrapper">
+                <label id="custom-btn" class="file-label" for="file-upload">Choose file</label>
```

Comment:
Here and in other places of this file. Rename ids to be REALLY unique, even in a future.

The simplest way: `name-input` -> `contact-form-name-input`